.github/workflows/typos.toml

```diff
@@ -0,0 +1,60 @@
+# See https://github.com/crate-ci/typos/blob/master/docs/reference.md
+# for an explanation of the available settings.
+
+[default]
+extend-ignore-identifiers-re = [
+   "OTSs", # mentioned in CHANGELOG ... ???
+]
+
+extend-ignore-re = [
```

Comment:
the more i read this file the less maintainable/practical it feels for me but i guess we can try it out. if it becomes a frequent occurrence to have to update this or i find myself not naming things the way i'd like to suite this thing though im going to advocate for giving it the boot :smile:  i am a high frequency spelling mistake maker though